api/core/v1alpha1/matrixonecluster_types.go

```diff
@@ -36,6 +36,10 @@ type MatrixOneClusterSpec struct {
 	// LogService is the default LogService pod set of this cluster
 	LogService LogSetBasic `json:"logService"`
 
+	// WebUI is the default web ui pod of this cluster
+	// +optional
+	WebUI WebUIBasic `json:"webui,omitempty"`
```

Comment:
optional field should be a pointer, otherwise omitempty would not work and it would be tricky to distinguish whether user request the WebUI